syntax = "proto3";

package fibonacci;

message FibonacciRequest
{
	uint32 value = 1;
}

message FibonacciResponse
{
	uint32 value = 1;
}

service FibonacciService
{
	rpc GetFibonacciSequence(FibonacciRequest) returns (stream FibonacciResponse) {}
}